src/tests/rules/valid-scripts.test.ts

```diff
@@ -0,0 +1,61 @@
+import { rule } from "../../rules/valid-scripts.js";
+import { ruleTester } from "./ruleTester.js";
+
+ruleTester.run("valid-scripts", rule, {
+	invalid: [
```

Comment:
[Testing] Some more invalid cases to think of:

* 1 incorrect value + 2 correct values
* 2 incorrect values
* duplicate values
* 2 incorrect values + 1 correct value